data-prepper-api/src/main/java/com/amazon/dataprepper/model/configuration/PluginSetting.java

```diff
@@ -136,6 +137,87 @@ public String getStringOrDefault(final String attribute, final String defaultVal
         throw new IllegalArgumentException(String.format(UNEXPECTED_ATTRIBUTE_TYPE_MSG, object.getClass(), attribute));
     }
 
+    /**
+     * Returns the value of the specified List<String>, or {@code defaultValue} if this settings contains no value for
+     * the attribute.
+     *
+     * @param attribute    name of the attribute
+     * @param defaultValue default value for the setting
+     * @return the value of the specified attribute, or {@code defaultValue} if this settings contains no value for
+     * the attribute
+     */
+    @SuppressWarnings("unchecked")
```

Comment:
It suppresses this warning from which only comes from Intellij `Unchecked cast: 'java.lang.Object' to 'java.util.Map<java.lang.String,java.util.List<java.lang.String>>' `. It is because of the cast on the return values from Object.